plugins/search-backend-module-elasticsearch/src/engines/ElasticSearchClientWrapper.ts

```diff
@@ -91,12 +91,23 @@ export class ElasticSearchClientWrapper {
   }
 
   search(options: { index: string | string[]; body: Object }) {
+    const searchOptions = {
+      ignore_unavailable: true,
+      allow_no_indices: true,
+    };
+
     if (this.openSearchClient) {
-      return this.openSearchClient.search(options);
+      return this.openSearchClient.search({
+        ...options,
+        ...searchOptions,
```

Comment:
Hi @drodil ! Thank you for this PR.
It first glance, I would argue that `options` should override `searchOptions`:
```
this.openSearchClient.search({
        ...searchOptions,
        ...options,
})
```

Did you had some use case in mind when you wrote it the other way?